src/parser/common.rs

```diff
@@ -384,6 +384,64 @@ mod tests {
 
     use super::{parse_matrix, parse_waveform_invocation};
 
+    /// A Quil program which uses all available instructions.
+    pub const KITCHEN_SINK_QUIL: &str = "DECLARE ro BIT[1]
```

Comment:
A+ variable naming here

Reply:
It's what I strive for. To be fair I did borrow it from somewhere in quil land